### PR TITLE
API-6365 Upgrades including Java 11

### PR DESCRIPTION
**Pull request by petekirby-ee**

### CODE REVIEW

File: app/models/file_upload/Nonce.scala [hunk 1]
```diff
@@ -27,6 +27,9 @@ case class Nonce(value: Int) {
       case nonce: Nonce => nonce.value == value
       case _            => false
     }
+
```
Comment (AndySpaven) [inline app/models/file_upload/Nonce.scala:30]: Any idea if this class even needs hashcodes and equals methods explicitly coding?

File: test/uk/gov/hmrc/gatekeepercomposeemailfrontend/mocks/config/FakeAppConfigImpl.scala [hunk 2]
```diff
@@ -30,10 +30,10 @@
 // * limitations under the License.
 // */
 //
-//package mocks.config
+//package uk.gov.hmrc.gatekeepercomposeemailfrontend.mocks.config
```
Comment (AndySpaven) [inline test/uk/gov/hmrc/gatekeepercomposeemailfrontend/mocks/config/FakeAppConfigImpl.scala:33]: Lots of commented out code and duplicate header

**Review by AndySpaven — DISMISSED** [review]
Just some comments to check

File: test/uk/gov/hmrc/gatekeepercomposeemailfrontend/mocks/config/FakeAppConfigImpl.scala (hunk 2 above)
Comment (petekirby-ee) [reply test/uk/gov/hmrc/gatekeepercomposeemailfrontend/mocks/config/FakeAppConfigImpl.scala]: Removed the file as the entire class was commented out.

File: app/models/file_upload/Nonce.scala (hunk 1 above)
Comment (petekirby-ee) [reply app/models/file_upload/Nonce.scala:30]: Not sure.  It was added by srinivasalumunagala last year.

**Review by AndySpaven — APPROVED** [review]

**Pull request merged**